<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>588</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Streak Tracking Program</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/Icon.webp</normaloff>../Images/Icon.webp</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-image: url(&quot;:/images/images/Morning.png&quot;);
	background-position: center;
    background-repeat: no-repeat;
    background-size: cover; /* Makes the image scale to fill the window */
    background-color: #f5f5f5;  /* Light gray background */
    font-family: 'Roboto', sans-serif;  /* Clean modern font */
    font-size: 14px;
    color: #333333;  /* Dark text color */
}

.QPushButton
{

color: #333;
border: 2px solid #555;
border-radius: 50px;


}

QPushButton#checkStreak
{
background-color: darkblue;
color:white;
font-size: 12px;
font-weight: bold;
border: 5px solid white;
}

QPushButton#checkStreak:hover , QPushButton#checkStreak:focus
{
background-color: black;
color: yellow;

}
QPushButton#checkStreak::pressed
{
background-color: red;
color: yellow;

}

QPushButton#logIn
{
background-color: green;
color:white;
font-size: 16px;
font-weight: bold;
border: 5px solid white;

}
QPushButton#logIn:hover , QPushButton#logIn:focus
{
background-color: black;
color: yellow;

}
QPushButton#logIn::pressed
{
background-color: red;
color: yellow;
}


QLCDNumber
{

    background-color: rgba(0, 0, 0, 1) ; /* Semi-transparent dark background */
    color: #ff4444; /* Bright red for numbers */
    font-size: 32px;
    font-weight: bold;
    border-radius: 12px;
    padding: 10px;
    text-align: center;

}

QListView
{
font-size:32px;
background-color:darkblue;
color:#F7FF00;
text-shadow:2px 2px 5px #FFF;
font-weight:900;



}

QMainWindow
{
background-color: rgba(255, 255, 255);
}

QMenuBar
{
background-color: rgba(255, 255, 255);
}

QStatusBar
{
background-color: rgba(255, 255, 255);
}

QCalendarWidget {
    background-color: #ffffff;  /* White background */
    border: 1px solid #cccccc; /* Light gray border */
    border-radius: 8px;
    padding: 5px;
}

QCalendarWidget QAbstractItemView {
    selection-background-color: #4caf50; /* Green for selected dates */
    selection-color: #ffffff; /* White text for selected dates */
}

QCalendarWidget QToolButton {
    background-color: #4caf50; /* Green buttons */
    color: #ffffff; /* White text */
    border-radius: 6px;
    padding: 5px;
}

QCalendarWidget QToolButton:hover {
    background-color: #388e3c; /* Darker green on hover */
}

QComboBox {
    background-color: #ffffff; /* White background */
    color: #333333; /* Dark gray text */
    border: 1px solid #cccccc; /* Light gray border */
    border-radius: 6px;
    padding: 5px;
}

QComboBox::drop-down {
    border-left: 1px solid #cccccc;
	background-color:purple;
}

QComboBox QAbstractItemView {
    selection-background-color: #4caf50; /* Green selection */
    selection-color: #ffffff; /* White text for selected item */
}



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="logList">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>331</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto,sans-serif</family>
      <pointsize>-1</pointsize>
      <weight>99</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">position:relative;
top:100px;
border-radius:25px;
border: solid 10px white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logIn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>420</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addRemove">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>AddRemove 
Tasks</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calender">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>392</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto,sans-serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:wheat;
border: solid 20px #333;
text-shadow: 5px 5px #333;
border-radius: 100px;</string>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QCalendarWidget::SingleSelection</enum>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="countDisplay">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;SansSerif&quot;;

</string>
    </property>
   </widget>
   <widget class="QComboBox" name="taskSelector">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto,sans-serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color : black;</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="backgroundImage.qrc"/>
 </resources>
 <connections/>
</ui>
